### PR TITLE
Update simulators to run on the Custom Docker Image

### PR DESCRIPTION
**Pull request by Pbarbosa92**
This PR makes the amrwind use the custom docker image.

### CODE REVIEW

**Review by bgregoinductiva — COMMENTED** [review]
What is the motivation for this change? Does the docs example need to be updated? Is it necessary to update the end to end test of this simulator?

**Comment by Pbarbosa92** [comment]
> What is the motivation for this change? Does the docs example need to be updated? Is it necessary to update the end to end test of this simulator?

The code works in the exact same way.

The point is to move all simulators to the "custom image". After that we can remove all executers from the task-runner, making the task runner simpler and making the process of adding new simulators easier.

**Comment by bgregoinductiva** [comment]
> > What is the motivation for this change? Does the docs example need to be updated? Is it necessary to update the end to end test of this simulator?
> 
> The code works in the exact same way.
> 
> The point is to move all simulators to the "custom image". After that we can remove all executers from the task-runner, making the task runner simpler and making the process of adding new simulators easier.

Ok... so why not change all the simulators now?

**Comment by Pbarbosa92** [comment]
> > > What is the motivation for this change? Does the docs example need to be updated? Is it necessary to update the end to end test of this simulator?
> > 
> > 
> > The code works in the exact same way.
> > The point is to move all simulators to the "custom image". After that we can remove all executers from the task-runner, making the task runner simpler and making the process of adding new simulators easier.
> 
> Ok... so why not change all the simulators already?

hmmm, I was going to do one PR per simulator but i can do them all in the PR.

**Comment by bgregoinductiva** [comment]
> > > > What is the motivation for this change? Does the docs example need to be updated? Is it necessary to update the end to end test of this simulator?
> > > 
> > > 
> > > The code works in the exact same way.
> > > The point is to move all simulators to the "custom image". After that we can remove all executers from the task-runner, making the task runner simpler and making the process of adding new simulators easier.
> > 
> > 
> > Ok... so why not change all the simulators already?
> 
> hmmm, I was going to do one PR per simulator but i can do them all in the PR.

If it's already tested I suggest to change all in the same PR.

**Comment by hpenedones** [comment]
Please update the PR title if now it will include multiple simulators in one go. Thanks!

File: examples/fvcom/fvcom.py [hunk 1]
```diff
@@ -17,8 +17,8 @@
 # Run simulation with config files in the input directory
 task = fvcom.run(input_dir=input_dir,
                  working_dir="run/",
-                 create_namelist="tst",
                  n_vcpus=1,
+                 debug=7,
```
Comment (bgregoinductiva) [inline examples/fvcom/fvcom.py:21]: Was this change intended?

**Review by bgregoinductiva — APPROVED** [review]

**Comment by Pbarbosa92** [comment]
@hpenedones This is a big PR.

After the reviews and everything can we do a small bugbash to test this ?

**Review by hpenedones — APPROVED** [review]
